test/test-platform-output.c

```diff
@@ -84,12 +84,78 @@ TEST_IMPL(platform_output) {
          (unsigned long long) rusage.ru_stime.tv_usec);
   printf("  page faults: %llu\n", (unsigned long long) rusage.ru_majflt);
   printf("  maximum resident set size: %llu\n",
-         (unsigned long long) rusage.ru_maxrss);
+         (unsigned long long)rusage.ru_maxrss);
 
   par = uv_available_parallelism();
   ASSERT_GE(par, 1);
   printf("uv_available_parallelism: %u\n", par);
 
+  #ifdef __linux__
```

Comment:
```suggestion
#ifdef __linux__
```
(don't indent preprocessor directives)